### ISSUE TITLE
IdnMappingGetAsciiTests.GetAscii failing on Unix in CI

### ISSUE THREAD

**Issue by stephentoub** [dotnet/corefx#8346]
This is consistently failing on OpenSUSE 13.2 and Ubuntu 16.04, e.g.
http://dotnet-ci.cloudapp.net/job/dotnet_corefx/job/master/job/opensuse13.2_debug_tst/248/consoleText
```
   System.Globalization.Tests.IdnMappingGetAsciiTests.GetAscii(unicode: ".", index: 0, count: 1, expected: ".") [FAIL]
      System.ArgumentException : Decoded string is not a valid IDN name.
      Parameter name: unicode
      Stack Trace:
            at System.Globalization.IdnMapping.GetAsciiCore(Char* unicode, Int32 count, UInt32 flags, Char* output, Int32 outputLength, Boolean reattempt)
            at System.Globalization.IdnMapping.GetAsciiCore(Char* unicode, Int32 count)
            at System.Globalization.IdnMapping.GetAscii(String unicode, Int32 index, Int32 count)
            at System.Globalization.IdnMapping.GetAscii(String unicode, Int32 index)
            at System.Globalization.IdnMapping.GetAscii(String unicode)
            at System.Globalization.Tests.IdnMappingGetAsciiTests.GetAscii(String unicode, Int32 index, Int32 count, String expected)
```
cc: @tarekgh, @hughbe

**Comment by hughbe** [comment]
Seems like Ubuntu 16.04 has different behaviour than 14.04 - we could delete this test if it's complicated, or include PlatformDetection

**Comment by tarekgh** [comment]
CC @ellismg

**Comment by tarekgh** [comment]
looks they have fixed the behavior mentioned in https://github.com/dotnet/corefx/issues/8242

```
GetAscii - single full stop (e.g.  . )
•Windows behaviour: throws an  ArgumentException  
•Mac OSX behaviour: throws an  ArgumentException  
•Unix behaviour: returns the original string
```

**Comment by hughbe** [comment]
I assume they've already fixed it but we use an old ICU on 14.04

**Comment by tarekgh** [comment]
This  should be fixed by the linked PR
